ow_sim_tests/test/terrain_interaction.py

```diff
@@ -176,6 +180,39 @@ def _assert_scoop_regolith_containment(self, assert_regolith_in_scoop=True):
               )
         )
 
+  """
+  Check a point inside a volume (dot product method)
+  """
+  def check_point_in_volume_dot(self, p):
```

Comment:
I'd like to see a function name here that has "dock" or "sample_dock" in it since the function is now specific to checking if a point is in the particular volume enclosed by the dock.